RemoteControl/RemoteControl.json

```diff
@@ -730,6 +781,106 @@
             "result": {
                 "$ref": "#/definitions/result"
             }
+        },
+        "startFirmwareUpdate": {
+            "summary": "Starts a firmware image update session for the specified remote.",
+            "events":{
+                "onFirmwareUpdateProgress" : "Generated at 0 and 100 percent and each time a download percent increment is reached."
+            },
+            "params": {
+                "type": "object",
+                "properties": {
+                    "macAddress": {
+                        "$ref": "#/definitions/macAddress"
+                    },
+                    "filename": {
+                        "summary": "The full path and filename for the firmware image.",
+                        "type": "string",
+                        "example": "/tmp/rcu_model_x_ver_a.tgz"
+                    }
+                },
+                "required": [
+                    "macAddress",
+                    "filename"
+                ]
+            },
+            "result": {
+                "type": "object",
+                "properties": {
+                    "status":{
+                        "type":"object",
+                        "properties":{
+                                "sessionId": {
+                                    "$ref": "#/definitions/sessionId"
+                                }
+                            },
+                            "required":[
+                               "sessionId"
+                            ]
+                    },
+                    "success": {
+                        "$ref": "#/definitions/success"
+                    }
+                },
+                "required":[
+                    "status",
+                    "success"
+                ]
+            }
+        },
+        "statusFirmwareUpdate": {
```

Comment:
Isn't the firmwareUpdateStatus part of the main status event? Do we need a separate call just for firmware update status?

Reply:
Right now, it's not a part of the main status event because i didn't want to clutter up the main status any further than it already is.  Also, I added a separate call because it's session based and there may be a use case where the app doesn't want to process events and instead just poll periodically until the session is completed.  However, I don't feel too strongly about this and we can omit the statusFirmwareUpdate call completely if we choose.
I considered adding an active session id field to the main status for each remote.  We may want to do this...

Reply:
I misunderstood the document structure. no comment here.